libs/application-generic/src/services/support.service.ts

```diff
@@ -0,0 +1,84 @@
+import { Logger } from '@nestjs/common';
+import { PlainClient, UpsertResult } from '@team-plain/typescript-sdk';
+
+const LOG_CONTEXT = 'SupportService';
+
+export class SupportService {
+  private plainClient: PlainClient;
+  constructor(private plainKey?: string | null) {}
+
+  async initialize() {
+    if (this.plainKey) {
+      this.plainClient = new PlainClient({
+        apiKey: this.plainKey,
+      });
+    } else {
+      Logger.error(
+        'Missing Plain API key: Unable to perform support operations. Please ensure the API key is set in the environment variables.',
+        LOG_CONTEXT,
+      );
+    }
+  }
```

Comment:
```suggestion
  constructor(private plainKey?: string | null) {
    if (this.plainKey) {
      this.plainClient = new PlainClient({
        apiKey: this.plainKey,
      });
    } else {
      Logger.error(
        'Missing Plain API key: Unable to perform support operations. Please ensure the API key is set in the environment variables.',
        LOG_CONTEXT,
      );
    }
  }
```

A small one, but I think we can get rid of the initialize method here as nothing here needs to happen async. Wdyt?